packages/cli/lib/resources/body-end.ejs

```diff
@@ -0,0 +1,18 @@
+<%= htmlWebpackPlugin.options.ssr() %>
```

Comment:
wonder if we could just do this in js instead of the template now

Reply:
I'm not sure if this is possible as `ejs` would escape anything we put in the directly.

Reply:
I meant not using `ejs` at all

Reply:
I'm a little worried that switching to no template engine may cause us more trouble than it's worth. We still have a bit of logic I'm both the `headEnd` and `bodyEnd` template that involves loops and if-statements. Many of those values are filled out by `HtmlWebpackPlugin` at runtime. I'm not sure how that would work, but I may be missing something.